### ISSUE TITLE
WIP-branch?

### ISSUE THREAD

**Issue by israelroldan** [dansup/bulma-templates#42]
I recently commented on #41 asking if there's any work in progress branch I can contribute to. I don't mind sending my contributions to the gh-branch, but I have read in several issue comments that you're working in several improvements to the project and I wouldn't want to step on your toes.

**Comment by dansup** [comment]
Hello @israelroldan, thanks for your interest. I have been fairly busy lately, I hope to push the WIP branch with a roadmap soon!

**Comment by dansup** [comment]
I have created a master branch for development, and will work towards releasing a 0.5.3 branch when all the templates are updated. In addition, I have updated the [project tracker](https://github.com/dansup/bulma-templates/projects/1) and will use that to organize the roadmap for the upcoming release.

The readme is updated with basic contribution instructions, I look forward to seeing your contributions!
